corehq/apps/smsbillables/management/commands/retrobill.py

```diff
@@ -1,39 +0,0 @@
-from corehq.apps.domain.models import Domain
```

Comment:
@nickpell this looks like it was a one-time old management command so i deleted it, let me know if that's not right